internal/pkg/service/common/etcdop/op/tracker.go

```diff
@@ -32,6 +33,8 @@ type TrackedOp struct {
 	Key []byte
 	// RangeEnd, optional, used if the WithPrefix or WithRange option is used
 	RangeEnd []byte
+	// KVs, optional, contains loaded KVs, only for Get operation
+	KVs []*mvccpb.KeyValue
 }
```

Comment:
There is a workaround, to block the Write Phase, if a key is delete from the tracked prefix.

**We can track all loaded keys.**